stl/inc/xhash

```diff
@@ -269,6 +269,12 @@ struct _Hash_vec {
             _Aliter_traits::max_size(_Mypair._Get_first()));
     }
 
+    _NODISCARD size_type capacity() const noexcept {
+        // This implementation never has capacity() differ from size(), but the previous implementation could.

```

Comment:
For the PR description and Changelog, do you have the VS update where we changed implementations (and thus mix-and-match became a compat bug)? I can look it up if you don't already have it.

Reply:
I do not. A quick peek at git blame suggests it was before we open-sourced, and I suspect your TFS-fu greatly outstrips mine. Would you mind digging?

Reply:
Found it! MSVC-PR-179257 merged 2019-05-08 introduced `_Hash_vec`. This shipped in VS 2019 16.3, Changelogged in ["C++20 Features and Fixes in VS 2019 16.1 through 16.6"](https://devblogs.microsoft.com/cppblog/c20-features-and-fixes-in-vs-2019-16-1-through-16-6/):

> The STL now uses allocator::construct for container user-supplied types only, so allocators that default-initialize rather than value-initialize no longer break the containers.